.github/semantic.yml

```diff
@@ -0,0 +1,14 @@
+# Always validate the PR title, and ignore the commits
+titleOnly: true
+# Always validate the PR title AND all the commits
```

Comment:
Oh, I guess we don't need these two lines if we're already using `titleOnly: true` correct?

Reply:
According to the [configuration](https://github.com/zeke/semantic-pull-requests#configuration) docs, all these settings are optional.
I personally prefer explicitly setting all the configuration settings.

I'll remove:
```diff
- titleAndCommits: false
```

Reply:
- [x] Fixed in 21b0609